test/rssignores.cpp

```diff
@@ -39,3 +43,219 @@ TEST_CASE(
 	REQUIRE(ignores.matches_resetunread("www.cool-website.com"));
 	REQUIRE_FALSE(ignores.matches_resetunread("www.smth.com"));
 }
+
+TEST_CASE("RssIgnores::handle_action() handles `ignore-article`",
```

Comment:
This only tests if it throws a `ConfigHandlerException` on invalid parameters.
It does not check if it actually handles the `ignore-article` when parameters are valid.

Reply:
I guess it is tested indirectly by further testcases so feel free to ignore.

Reply:
I think it makes sense to test directly, too. Please take a look at https://github.com/newsboat/newsboat/pull/850/commits/20c1214fc639202e38715399fc3d3509ec2d730c.

Reply:
I'll take the :+1: on my previous comment as a sign that you've taken a look and think the changes are fine :)